ethcontract-mock/Cargo.toml

```diff
@@ -14,7 +14,7 @@ Tools for mocking ethereum contracts.
 [dependencies]
 ethcontract = { version = "0.22.0", path = "../ethcontract", default-features = false, features = ["derive"] }
 hex = "0.4"
-mockall = "0.11"
+mockall = "0.11.2"
```

Comment:
This seems like an unrelated change... Is there a reason we need at least `0.11.2`?